commands/help.js

```diff
@@ -1,10 +1,33 @@
+const fs = require('fs');
+const discord = require('discord.js');
+
+// Dynamic Help command
+
 module.exports.run = async (client, message, args, prefix) => {
-    message.channel.sendMessage('test!')
-    .then().catch(console.error);
+    // Setup embed
+    const helpEmbed = new discord.RichEmbed()
+    .setTitle("Command Help list")
+    .setThumbnail("https://images.emojiterra.com/twitter/v12/512px/1f5a5.png")
+    .setColor("#00FFFF")
+
+    // Gather all command names and description
+    let commandFiles = fs.readdirSync('./commands').filter(file => file.endsWith('.js'))
```

Comment:
I think we can avoid re-parsing `./commands`. We already register this info in discord.js so we could do something like
```js
    let items = [];
    client.commands.forEach(element => {
        let name = element.help.name;
        let desc = element.help.description;
        let dev = element.help.dev;
        // ...
    });
```